src/components/shared/custom-modal-window/modal-window.tsx

```diff
@@ -0,0 +1,32 @@
+import styles from './style.module.scss';
+import { CustomInput } from '../custom-input';
+import { CustomButtonComponent } from '../custom-button';
+import { Box, Modal } from '@mui/material';
+import { useState } from 'react';
+
+export const ModalWindow = () =>{
```

Comment:
add type. export const ModalWindow: FC = () =>{}

Reply:
use https://mui.com/material-ui/react-dialog/ instead of modal. rename the component to CustomModal and use <Dialog /> from a link